autosklearn/metrics/__init__.py

```diff
@@ -32,11 +33,13 @@ def __init__(
         worst_possible_result: float,
         sign: float,
         kwargs: Any,
+        needs_x: bool = False,
```

Comment:
I think we should use an upper-case X here to indicate that this is a 2d-array of samples and not a single sample.

Reply:
done